libsql-replication/src/replicator.rs

```diff
@@ -208,7 +215,7 @@ where
 
     pub async fn try_perform_handshake(&mut self) -> Result<(), Error> {
```

Comment:
Will it be helpful if we will expose metric about actual amount of attempts?

Reply:
Possibly but there are two cases here: embedded replicas and regular replicas. If we decide to add such metric it should be a separate PR anyways.

Reply:
Ok...